packages/design-picker/src/types.ts

```diff
@@ -60,8 +60,11 @@ export interface StyleVariationStylesColor {
 export interface DesignRecipe {
 	stylesheet?: string;
 	pattern_ids?: number[] | string[];
+	pattern_html?: string;
```

Comment:
This is interesting... does it mean we don't need `pattern_ids`, `header_pattern_ids`, and `footer_pattern_ids` anymore? I think we have no use cases that are still using them.

Reply:
We still need the `pattern_ids`, `header_pattern_ids` and `footer_pattern_ids` for the mshot preview

Reply:
Ah.... you're right 😅 thanks 😄